foca/models/config.py

```diff
@@ -511,14 +514,24 @@ class MongoConfig(FOCABaseConfig):
     Connexion app.
 
     Args:
+        username: The username required for database access, in case
+        authorization is enabled.
+        password: The passwrod required for database access, in case
+        authorization is enabled.
         host: Host at which the database is exposed.
         port: Port at which the database is exposed.
+        db_name: The database name.
```

Comment:
Database names are already taken care off deeper down the model structure, under `dbs`, with support for multiple databases. I wouldn't want to change that up again.